Extensions/retags.js

```diff
@@ -102,14 +105,13 @@ XKit.extensions.retags = {
 				   $target = $element;
 				   url = $target.find('.action').data('post-url');
 			// Activity
-			} else if ($element.hasClass('ui_note')) {
-				if ($element.find('.part_response').length) {
-					$element.addClass('is_response');
-				}
+			} else if ($element.hasClass('activity-notification')) {
 				retagClass = 'is_retags';
 				$target = $element;
-				url = $target.find('.part_glass').attr('href');
-			// dashboard
+				var glass = $($element).find('.activity-notification__glass');
```

Comment:
Yeah, I guess you're right. idk why but it felt nicer to me to use the individual data since it was there but I guess making the branches consistent is better in the long run. Will update that as well.